lib/membrane/video_compositor/implementations/wgpu/wgpu.ex

```diff
@@ -0,0 +1,41 @@
+defmodule Membrane.VideoCompositor.Wgpu do
+  @moduledoc """
+  This module implements video composition in wgpu
+  """
+  @behaviour Membrane.VideoCompositor.FrameCompositor
+
+  alias Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.Position
+  alias Membrane.VideoCompositor.Implementations.OpenGL.Native.Rust.RawVideo
+  alias Membrane.VideoCompositor.Implementations.Wgpu.Native
+
+  @impl true
+  def init(output_caps) do
+    {:ok, output_caps} = RawVideo.from_membrane_raw_video(output_caps)
+    Native.init(output_caps)
+  end
+
+  @impl true
+  def merge_frames(frames, internal_state) do
```

Comment:
Update `internal_state` parameter position. It is the first parameter in almost all callbacks from the `FrameCompositor` behavior. You should rebase from the master.